src/main/kotlin/mods/eln/craft/CraftingRecipes.kt

```diff
@@ -116,6 +118,14 @@ object CraftingRecipes {
         recipeECoal()
 
         recipeGridDevices(Eln.oreNames)
+        recipeMaceratorModOres()
+        craftBrush()
+        val cal: Calendar = Calendar.getInstance()
+        val month: Int = cal.get(Calendar.MONTH) + 1
+        val day: Int = cal.get(Calendar.DAY_OF_MONTH)
+        if(month == 12 && day == 25) {
+            recipeChristmas()
+        }
```

Comment:
Heh. Probably the whole month would be okay. I spend a lot of time celebrating it with friends in the days up to it but generally not on that day because I'm with family. I'm sure others are the same way.

Reply:
I'm sorry, I don't know much about Christmas and I'm not sure when to open the recipes of Christmas things

Reply:
All good. I can make a small edit to support the whole month of December.